client/stylesheets/_toCat.scss

```diff
@@ -5171,6 +5178,7 @@ On click the images will be displayed at normal size to complete the effect
     width: 100%;
     margin-top: 23px;
     margin-bottom: 200px;
+    display: inline-block !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces